modules/dnn/src/ocl4dnn/src/ocl4dnn_conv_spatial.cpp

```diff
@@ -155,8 +155,8 @@ OCL4DNNConvSpatial<Dtype>::OCL4DNNConvSpatial(OCL4DNNConvConfig config)
     // assumption: spatial dimension is 2.
     kernel_h_ = config.kernel.height;
     kernel_w_ = config.kernel.width;
-    pad_h_ = config.pad.height;
-    pad_w_ = config.pad.width;
+    pad_t_ = config.pad_t;
+    pad_l_ = config.pad_l;
```

Comment:
I think we can keep convolution kernel without changes.

Reply:
done